### ISSUE TITLE
Podcast Player: Focus Styles Missing or Cutoff

### ISSUE THREAD

**Issue by jeryj** [Automattic/jetpack#15537]
The podcast player buttons (play/pause and mute/unmute) do not have any focus styles. They used to have a dotted outline, but using `mask` in the CSS prevents this outline from showing up.

To meet WCAG guidelines, we need to have a focus indicator present. Here's an example of the failure from the docs: https://www.w3.org/WAI/WCAG21/Techniques/failures/F78

Fixes needed:
- MEjs player button focus styles
- In the editor canvas, we need a focus indicator on the tracks.
- The focus on tracks on the front-end is slightly cut-off due to `overflow: hidden;` on the wrapper.

#### Steps to reproduce the issue

**Front-end**
1. Go to the front-end of a podcast player
2. Tab through the player and see that there is no focus indicator on the play/pause or sound icons
3. See the tracklist focus cutoff

**Editor**
1. Go to the editor of a podcast player
2. Tab through the player and see that there is no focus indicator on the tracklist items. I think this depends on the theme. Some use `box-shadow` as a focus indicator, and we're resetting that for consistency. Maybe we should remove the line that removes `:focus` `box-shadow`.

#### What I expected

Focus indicators should be present and clearly communicated without anything being cutoff. 
#### What happened instead

**Screenshots**

<img width="617" alt="Screen Shot 2020-04-23 at 2 00 07 PM" src="https://user-images.githubusercontent.com/967608/80138699-cff21d80-856a-11ea-9429-6c6071cf56bc.png">

<img width="607" alt="Screen Shot 2020-04-23 at 1 59 56 PM" src="https://user-images.githubusercontent.com/967608/80138696-cec0f080-856a-11ea-81fb-c0a979d2301c.png">
